code/game/objects/structures/statues.dm

```diff
@@ -293,10 +253,210 @@
 ///////////////////////////////bronze///////////////////////////////////
 
 /obj/structure/statue/bronze
-	material_drop_type = /obj/item/stack/tile/bronze
+	custom_materials = list(/datum/material/bronze=MINERAL_MATERIAL_AMOUNT*5)
+	abstract_type = /obj/structure/statue/bronze
 
 /obj/structure/statue/bronze/marx
 	name = "\improper Karl Marx bust"
 	desc = "A bust depicting a certain 19th century economist. You get the feeling a specter is haunting the station."
 	icon_state = "marx"
 	art_type = /datum/component/art/rev
+
+
+/obj/item/chisel
+	name = "chisel"
+	desc = "breaking and making art since 4000 BC"
+	icon = 'icons/obj/statue.dmi'
+	icon_state = "chisel"
+	inhand_icon_state = "screwdriver_nuke"
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	flags_1 = CONDUCT_1
+	slot_flags = ITEM_SLOT_BELT
+	force = 5
+	w_class = WEIGHT_CLASS_TINY
+	throwforce = 5
+	throw_speed = 3
+	throw_range = 5
+	custom_materials = list(/datum/material/iron=75)
+	attack_verb_continuous = list("stabs")
+	attack_verb_simple = list("stab")
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	usesound = list('sound/items/screwdriver.ogg', 'sound/items/screwdriver2.ogg')
+	drop_sound = 'sound/items/handling/screwdriver_drop.ogg'
+	pickup_sound =  'sound/items/handling/screwdriver_pickup.ogg'
+	item_flags = EYE_STAB
+	sharpness = SHARP_POINTY
+
+	/// Block we're currently carving in
+	var/obj/structure/carving_block/prepared_block
+	/// The thing it will look like
+	var/atom/movable/current_target
+	/// Currently chosen preset statue type
+	var/current_preset_type
+	/// If tracked user moves we stop sculpting
+	var/mob/living/tracked_user
+	/// Currently sculpting
+	var/sculpting = FALSE
+/*
```

Comment:
```suggestion
	/// Block we're currently carving in
	var/obj/structure/carving_block/prepared_block
	/// The thing it will look like
	var/atom/movable/current_target
	/// Currently chosen preset statue type
	var/current_preset_type
	/// If tracked user moves we stop sculpting
	var/mob/living/tracked_user
	/// Currently sculpting
	var/sculpting = FALSE


/obj/item/chisel/Destroy()
	prepared_block = null
	current_target = null
	tracked_user = null
	return ..()


/*
```
Might want to register for their destruction as well, and handle it, if the variables are not held for a very short time.